### PR TITLE
[TRAFODION-1893]ODBC:Remove vcredist from win-odbc build

### PR DESCRIPTION
**Pull request by rwbzx**
1. Remove vcredist_x64.exe packaging inside win-odbc installer.
2. If no install of dependency package is detected, a message-box will popup prompting users to download and install it. 
3. And by the time the [OK] button of the prompt message-box is clicked, the installer will use system-default browser to open download URL, also at the same time, the Setup Finish Page will pop-up.

### CODE REVIEW

**Comment by xwq** [comment]
@svarnau 
Hi Steve, could you please add him as an user.

**Comment by svarnau** [comment]
jenkins, add user

File: win-odbc64/Install/win64_installer/installer.iss [hunk 1]
```diff
@@ -152,6 +141,40 @@ begin
...
+  iQuotePos: Integer;
+  lLength: Longint;
+  iRetCode: Integer;
+begin
+  if CurStep=ssPostInstall then
+  begin
+    bRedistNeedsInstall := VCRedistNeedsInstall();
+    if bRedistNeedsInstall=True then
+    begin
+      MsgBox('Install of Visual C++ Redistributable Packages for Visual Studio 2013 is not dectected.'+ #13#10#13#10 + 'Please download vcredist_x64.exe from Microsoft Download Centre and install it.', mbInformation, MB_OK);
```
Comment (hegdean) [inline win-odbc64/Install/win64_installer/installer.iss]: Can we change the message to be as follows:
Run-time dependency needed by the driver is not found. Proceeding to download and install run-time dependency from Microsoft.

**Comment by hegdean** [comment]
Please change the version of the driver from 1.0 to 2.1 wherever applicable

**Comment by hegdean** [comment]
Change looks good to me.

**Comment by svarnau** [comment]
Looks good to me too.

**Pull request merged**